test/net_http_test.rb

```diff
@@ -312,6 +312,43 @@ def test_adding_custom_errors_do_trip_circuit
     end
   end
 
+  def test_5xxs_trip_circuit_when_fatal_server_flag_enabled
+    options = proc do |host, port|
+      {
+        tickets: 2,
```

Comment:
I'd consider disabling tickets for this test, since you're not testing them. You can do this by setting `tickets: false`.